Modules/Misc/srum-dump.mkape

```diff
@@ -1,14 +1,15 @@
 Description: 'SRUM-dump: Dump contents of the srum database'
 Category: SystemActivity
-Author: Brian Maloney
-Version: 1.0	
-Id: 8e0a2314-79e4-48f2-81ad-a0719e8af680
+Original Author: Brian Maloney
+Author: Jay Houlden
```

Comment:
Can you please add both authors on the same line? Just do Author/Author or some variation thereof.

Reply:
@theredmoose not sure if you missed this one or not. Once you fix this, I can merge.